### ISSUE TITLE
npm ERR! after updating to cordova@8.1.0 

### ISSUE THREAD

**Issue by bnfrnz** [apache/cordova-cli#338]
After updating to `cordova@8.1.0` I'm getting the following errors:
```
npm ERR! missing: inquirer@^0.10.0, required by insight@0.8.4
npm ERR! missing: exit-hook@^1.0.0, required by restore-cursor@1.0.1
```
After downgrading to `cordova@8.0.0` again, the errors disappear.

**Comment by brodybits** [comment]
Can you please share the following information:
- which platform
- output of `node --version`
- output of `npm --version`

I just tried Cordova CLI 8.1.0 on Node.js 4, 6, 8, and 10 on my Windows PC. I found issue on Node.js 4 and raised issue #339; `cordova --version` on Cordova CLI 8.1.0 works fine for me on Node.js 6, 8, and 10.

Possibly related: #337

**Comment by bnfrnz** [comment]
Sure, here the versions info: 
macOS 10.13.6
Node.js 8.12.0 LTS
npm 6.4.1

Run `npm list -g --depth=0` to confirm the above ERR! logs.

I have `cordova` installed globally while `cordova-ios` and `cordova-android` are installed on project level. I can't imagine this being an issue here but thought I'd mention it just in case.

**Comment by shazron** [comment]
FWIW I was successful in installing Cordova@8.10 using a clean VM as well as my dev machine:
```
node 10.9.0
npm 6.2.0
macOS 10.13.6
```
and also 
```
node 8.12.0
npm 6.4.1
macOS 10.13.6
```

**Comment by bnfrnz** [comment]
Sorry, just to be clear: my install went through clean as well. That's not the issue. The npm errors only surfaced after, upon running `npm list -g`. These two packages are some lower level dependent modules of `cordova-ios`. I'm not sure what they do or if they're still being used, and whether or not you'll run into any issues. That may depend on your project.

My ionic cordova build ran into trouble. I don't have time for debugging right now so I just downgraded to `cordova@8.0.0` again which runs smoothly.

I had read the [release notes](https://cordova.apache.org/news/2018/09/26/cordova-cli-8.1.0.html) and suspect these npm ERR! logs are somehow related to [#312](https://github.com/apache/cordova-cli/pull/312).

**Comment by shazron** [comment]
ah yes, I can repro now.

**Comment by raphinesse** [comment]
These error logs are most likely a result of our fiddling with transitive dependencies to resolve npm audit warnings. By themselves, they are OK and should not worry us.

Unfortunately we also messed up functionality when updating the dependencies (see #337)

**Comment by raphinesse** [comment]
I'll close this as it's not really an issue. Nevertheless, the errors during `npm list -g` will probably be gone when we fixed #337.
